packages/Webkul/API/Http/Resources/Catalog/Product.php

```diff
@@ -56,6 +56,7 @@ public function toArray($request)
                     $product->getTypeInstance()->haveSpecialPrice(),
                     core()->currency($product->getTypeInstance()->getSpecialPrice())
                 ),
+            $product->getTypeInstance()->getProductPrices(),
```

Comment:
Hi,

If you check the code you will see there are `price`, `formatted_price`, etc. are present may  I know what is the need of this again even without any keys.

Reply:
At least, if needed please add the key to it.

Reply:
Hello Devansh,

Thanks for reviewing. I have added `prices` key to the resource.

The need is that when a product have special price only the special price is returned in the response, thus we don't have the original price in the response.